databricks/koalas/namespace.py

```diff
@@ -1635,6 +1626,23 @@ def _to_datetime2(arg_year, arg_month, arg_day,
         format=format,
         infer_datetime_format=infer_datetime_format)
 
+    index_map = None  # type: Optional[List[IndexMap]]
+
+
+def _get_index_map(sdf, index_col=None):
```

Comment:
Could you add type hints?

Reply:
Then I think we don't need the type hints at each returned place above.

Reply:
Thanks for comment! I've fixed them. and remove unnecessary type hints above